devito/interfaces.py

```diff
@@ -151,7 +152,10 @@ def _indices(cls, **kwargs):
                 raise ValueError("Unknown symbol dimensions or shape")
             _indices = [x, y, z]
             shape = kwargs.get('shape')
-            dimensions = _indices[:len(shape)]
+            if len(shape) <= 3:
+                dimensions = _indices[:len(shape)]
+            else:
+                dimensions = [symbols("x%d" % i) for i in range(1, len(shape) + 1)]
```

Comment:
Why are we creating these `xi` indices here? I don't think the symbolic objects should know anything about the temporary variable indices created in the Propagator.